addons/mail/models/mail_channel.py

```diff
@@ -413,7 +413,7 @@ def message_post(self, *, message_type='notification', **kwargs):
 
         self.filtered(lambda channel: channel.is_chat).mapped('channel_last_seen_partner_ids').sudo().write({'is_pinned': True})
 
-        message = super(Channel, self.with_context(mail_create_nosubscribe=True)).message_post(message_type=message_type, moderation_status=moderation_status, **kwargs)
+        message = super(Channel, self.with_context(mail_create_nosubscribe=True, mail_post_autofollow=False)).message_post(message_type=message_type, moderation_status=moderation_status, **kwargs)
```

Comment:
The PR is re-introducing the possibility to mention in channels, or more precisely to actually forward the mentioned persons as `partner_ids` to `message_post` so that they get a notification. The problem is that when we post a comment (as opposed to a note) we have to set `mail_post_autofollow` `True` (because of the chatter), but we actually never want to add followers to a channel, and especially not automatically.

Reply:
I'll add comment.